entries/b/bell.ca.json

```
@@ -0,0 +1,15 @@
+{
+  "Bell": {
+    "domain": "bell.ca",
+    "contact": {
+      "facebook": "BellCanada",
+      "twitter": "Bell"
```

Support accounts are preferred as they are actively monitored by the company.
```suggestion
      "twitter": "Bell_Support"
```